content/docs-v1/zooming-images.md

```diff
@@ -57,21 +57,41 @@ To speed development time, Quire will skip processing zooming images that have p
 
 Some basic tips for now include: 
 
 - Optimize your source images. The larger the file size in megabytes, the more time it will take to process when your project is building. 
-- Don't use too many on one page or in one lightbox
+- Don't use too many images on one page or in one lightbox
 - Use placeholder images until ready to build the actual work 
 - Host images externally (see the following section)
 
+## Add Existing External IIIF Images
+
+In addition to creating deep zooming images in your Quire project by adding `zoom: true` to a `figures.yaml` entry, you can also use existing external IIIF images. This is achieved by adding a few additional attributes to the `figures.yaml` entry, including `manifestId`, `canvasId`, and `printImage`. 
+
+```yaml
+- id: "irises"    
+  manifestId: https://media.getty.edu/iiif/manifest/53be857e-41e8-4198-b45d-2e0f52d3051b
+  canvasId: https://media.getty.edu/iiif/manifest/canvas/e5d29650-11f8-4897-9540-54a9dd65b04f.json
+  printImage: figures/irises.jpg
+  src: figures/irises.jpg 
+```
+
+The `manifestId` or manifest URL is a link to a JSON file that contains important metadata about an object including things like artist, name, and copyright information. A manifest URL will always accompany a IIIF image, however, each institution displays this information a bit differently. Sometimes there is a IIIF button that will take you to a manifest, other institutions might include the manifest URL below or alongside the tombstone information on an object's collection page. Here is an example of where the manifest URL is found on the Getty collection page for Van Gogh's *Irises*:
+
+{{< q-figure id="iiif-manifest" >}}
+
+The next attribute you need to include in the `figures.yaml` is the `canvasId`. Every institution's manifest is formatted differently so locating the `canvasID` can be a bit tricky. The easiest way to find it is with a tool like Liz Fischer's [IIIF Manifest Explorer](https://www.lizmfischer.com/iiif-tools/manifest). If you plug the manifest URL (`manifestId`) into the URL bar, the tool will breakdown all the metadata in the manifest so it is easier to read, including identifying the `canvasID` which you will then add to your `figures.yaml` entry.
+
+{{< q-figure id="liz-fischer" >}}
 
-## Use Existing External IIIF Images
+In the `figures.yaml` you will also need to include a `printImage` attribute and a `src` attribute. These point to where you have saved your image in your project. So for the example above, `irises.jpg` is saved in the projects `_assets/images/figures` folder. The `src` value will find the image and add it to your Markdown page and the `printImage` will make sure that image is included in the PDF/print version of your publication. 
```

Comment:
I'm not sure what you mean by "The `src` value will find the image and add it to your Markdown page."?

Also, should we acknowledge that it's sort of silly to have to point to the static images twice, but that we'd like to fix it someday?

Also, for people who know IIIF, should we point out that they can't use an external URL for either `src` or `printImage` (like https://media.getty.edu/iiif/image/e5d29650-11f8-4897-9540-54a9dd65b04f/full/900,/0/default.jpg)? This is the issue that Rachel at Nelson-Atkins ran into, I think.